src/pages/workspaces/workspace/Data.js

```diff
@@ -105,6 +106,19 @@ const DataTypeButton = ({ selected, children, iconName = 'listAlt', iconSize = 1
   ])
 }
 
+const DataImportPlaceholder = () => {
+  return div({
+    style: { ...Style.navList.item(false), color: colors.dark(0.7) }
+  }, [
+    div({ style: { flex: 'none', display: 'flex', width: '1.5rem' } }, [
+      icon('downloadLight', { size: 14, style: { color: 'green' } })
```

Comment:
The icon in the screenshot looked green. Not sure the exact value I should use. I'll check with Joy.

Reply:
I had a thought that maybe this should be `colors.primary()`.